Update KEYS.md to use a new dumping guide and correct title.key formatting (#814)
Ryujinx/Ryujinx: KEYS.md

Keys are required for decrypting most of the file formats used by the Nintendo Switch.

 Keysets are stored as text files. These 2 filenames are automatically read:  
* `prod.keys` - Contains common keys used by all Nintendo Switch devices.
* `title.keys` - Contains game-specific keys.

Ryujinx will first look for keys in `RyuFS/system`, and if it doesn't find any there it will look in `$HOME/.switch`.
To dump your `prod.keys` and `title.keys` please follow these following steps.
1.	First off learn how to boot into RCM mode and inject payloads if you haven't already. This can be done [here](https://nh-server.github.io/switch-guide/).
2.	Make sure you have an SD card with the latest release of [Atmosphere](https://github.com/Atmosphere-NX/Atmosphere/releases) inserted into your Nintendo Switch.
3.	Download the latest release of [Lockpick_RCM](https://github.com/shchmue/Lockpick_RCM/releases).
4.	Boot into RCM mode.

> 5.	Inject the `Lockpick_RCM.bin` that you have downloaded at `Step 3.` using your preferred payload injector. We recommend [TegraRCMGUI](https://github.com/eliboa/TegraRcmGUI/releases) as it easy to use and has a decent feature set.

Reviewer: ```suggestion
5.	Inject the `Lockpick_RCM.bin` that you have downloaded at `Step 3.` using your preferred payload injector. We recommend [TegraRCMGUI](https://github.com/eliboa/TegraRcmGUI/releases) as it is easy to use and has a decent feature set.
```

Missing the word `is` here!

Keys are required for decrypting most of the file formats used by the Nintendo Switch.

> Keysets are stored as text files. These 3 filenames are automatically read:  

Reviewer 2: ```suggestion
 Keysets are stored as text files. These 2 filenames are automatically read:  
```

> `title.keys` - Contains game-specific keys.

Reviewer 2: Maybe we can add dot here:

- `prod.keys` - Contains common keys usedy by all Switch devices.  
- `console.keys` - Contains console-unique keys.  
- `title.keys` - Contains game-specific keys.

Ryujinx will first look for keys in `RyuFS/system`, and if it doesn't find any there it will look in `$HOME/.switch`.
To dump your `prod.keys` and `title.keys` please follow these following steps.

> 1.	First off learn how to boot into RCM mode and inject payloads if you haven't already. This can be done [here](https://switch.homebrew.guide/).

Reviewer 2: In `Readme` we use https://nh-server.github.io/switch-guide/ link, I'm not against your guide, but we have to use the same guide link everywhere for consistency.

> 2.	Make sure you have an SD card with the latest release of [https://github.com/AtlasNX/Kosmos/releases](https://github.com/AtlasNX/Kosmos/releases) inserted into your Nintendo Switch.

Reviewer 2: As we use a RCM payload, why AtlasNX is needed ? You can use the payload with Atmosphère or Hekate on the console too... So I think this line can be removed.

Author: For Firmware 7.x and above, we'll need sept from Atmosphere. I agree I shouldn't have used Kosmos, but Atmosphere is the minimum we'll need.

Reviewer 2: Oh right, 7.x+ needs TSEC to compute keys. Then tell users they need AMS, since it’s the minimum ;)!

3.	Download the latest release of [Lockpick_RCM](https://github.com/shchmue/Lockpick_RCM/releases).
4.	Boot into RCM mode.

> 5.	Inject the `Lockpick_RCM.bin` that you have downloaded at `Step 3.` using your preffered payload injector. We recommend [TegraRCMGUI](https://github.com/eliboa/TegraRcmGUI/releases) as it easy to use and has cool feautures.

Reviewer: ```suggestion
5.	Inject the `Lockpick_RCM.bin` that you have downloaded at `Step 3.` using your preferred payload injector. We recommend [TegraRCMGUI](https://github.com/eliboa/TegraRcmGUI/releases) as it easy to use and has a decent feature set.
```

For example:

```

> 01000000000100000000000000000003 = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

Reviewer 2: at line 50:

>Title keys are stored in the format `rights_id,key`.

You should change that too since you change the example.
